dev-lua/lua-argparse/lua-argparse-0.6.0.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit toolchain-funcs
+
+DESCRIPTION="Command line argument parser for the Lua Programming Language"
+HOMEPAGE="https://github.com/mpeterv/argparse"
+SRC_URI="https://github.com/mpeterv/argparse/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc luajit test"
+
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	virtual/pkgconfig
+	doc? ( dev-python/sphinx )
+	test? ( dev-lua/busted )"
```

Comment:
it should be 
```
test? (
    ${RDEPEND}
    dev-lua/busted
)
```

with RDEPEND moved above BDEPEND declaration of course.
you still need runtime deps to run the test most likely.

Reply:
fixed

Reply:
fixed